packages/retail-ui/components/Tooltip/Tooltip.tsx

```diff
@@ -164,6 +164,12 @@ class Tooltip extends React.Component<TooltipProps, TooltipState> {
       this.forceUpdate();
     }
   }
+  
+  componentWillReceiveProps(nextProps: TooltipProps) {
+    if ((this.props.trigger === 'hover' || this.props.trigger === 'hoverAnchor') && nextProps.trigger === 'closed') {
```

Comment:
Тут кажется можно упростить до
```suggestion
    if (nextProps.trigger === 'closed') {
```

Потому что с `focus` и `click` триггерами та же беда может происходить